src/tlitest/tlitest-run

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+cd $(dirname $0)
+echo "RUN_PATH=$PWD"
+
+TEST_DIRS="
+../../lib
+/usr/local/lib
+/lib
+"
```

Comment:
We don't really know where we'll be installed. To be handled reliably configure would need to substitute its install location here. However, we don't need to handle this now, if ever. We already ensured our executable names are globally-unique, and that's enough for now.

Reply:
Ok.